shoop/simple_pricing/module.py

```diff
@@ -29,50 +31,52 @@ def get_context_from_request(self, request):
         customer = getattr(request, "customer", None)
         shop = getattr(request, "shop", None)
 
+        assert shop is not None
```

Comment:
@akx: Do you mean that it should rather use just
```python
shop = request.shop
```
?

Though, I would still add an assert, but maybe something like
```python
assert isinstance(shop, shoop.core.models.Shop)
```
would be more useful...